fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/exceptionhandlers/ItemNotFoundExceptionMapper.java

```diff
@@ -40,8 +41,6 @@
 
     @Override
     public Response toResponse(final ItemNotFoundException e) {
-
-        LOGGER.debug("Exception intercepted by ItemNotFoundExceptionMapper: {}\n", e.getMessage());
```

Comment:
Is there a rationale for which exception-mappers now have `warn` exceptions instead of calling `debugException`, and which mappers simply remove the current `debug` logging?

Reply:
Yep, see my previous comment for the rational. Additionally, I removed some duplicate debug logs, like the one commented on here. There's no reason to both have a debug call in this class and then call debugException as well.